cmake/ctest/drivers/atdm/toss3/drivers/Trilinos-atdm-toss3-intel-debug-openmp-panzer.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+export SRUN_CTEST_TIME_LIMIT_MINUTES=300 # 5 hour time limit
+$WORKSPACE/Trilinos/cmake/ctest/drivers/atdm/toss3/local-driver.sh
```

Comment:
@fryeguy52, for now, I think we need to disable the running of the tests on 'toss3' ('serrano' and 'chama') for with:

```
export CTEST_DO_TEST=OFF
```

right?

I will get the ctest driver working to run the tests on the compute node separately later.

Reply:
export CTEST_DO_TEST=OFF 
is in the $WORKSPACE/Trilinos/cmake/ctest/drivers/atdm/toss3/local-driver.sh script.  Is it preferable to move it the individual scripts?

Reply:
Okay, I see it set there now.  That is a much better place for for this.  Thanks!